Source/WebCore/rendering/style/StyleCachedImage.cpp

```diff
@@ -68,7 +73,7 @@ bool StyleCachedImage::equals(const StyleCachedImage& other) const
         return false;
     if (m_cssValue.ptr() == other.m_cssValue.ptr() || m_cssValue->equals(other.m_cssValue.get()))
         return true;
-    if (m_cachedImage && m_cachedImage == other.m_cachedImage)
+    if (cachedImage() == other.cachedImage())
```

Comment:
Do you want both nulls to compare as equal? That's a behavior change.

Reply:
I will revert this change.